src/components/NWCBudgetEditor.tsx

```diff
@@ -31,29 +31,49 @@ export type BudgetForm = {
 export function NWCBudgetEditor(props: {
     initialProfile?: NwcProfile;
     initialName?: string;
+    initialAmount?: string;
+    initialInterval?: "Day" | "Week" | "Month" | "Year";
     onSave: (value: BudgetForm) => Promise<void>;
 }) {
     const i18n = useI18n();
 
+    console.log(props);
+
+    const connection_name = props.initialProfile?.name
+        ? props.initialProfile?.name
+        : props.initialName
+        ? props.initialName
+        : "";
+
+    // If there's an initial profile, look at that, otherwise default to false
+    const auto_approve = props.initialProfile?.require_approval
+        ? !props.initialProfile?.require_approval
+        : false;
```

Comment:
Think this can just be `props.initialProfile?.require_approval || false`

Reply:
no there's a weird double negative here